```
Error: cmd/notification.go:1108:29: func `(*NotificationSys).getOnlinePeers` is unused (unused)
func (sys *NotificationSys) getOnlinePeers() []*peerRESTClient {
                            ^
make: *** [Makefile:41: lint] Error 1
Error: Process completed with exit code 2.
```

atleast proves the bug :)

--- a/cmd/notification.go
+++ b/cmd/notification.go
@@ -1104,17 +1104,6 @@ func (sys *NotificationSys) ServerInfo(metrics bool) []madmin.ServerProperties {
 	return reply
 }
 
-// returns all the peers that are currently online.
-func (sys *NotificationSys) getOnlinePeers() []*peerRESTClient {
-	var peerClients []*peerRESTClient
-	for _, peerClient := range sys.allPeerClients {
-		if peerClient != nil && peerClient.IsOnline() {
-			peerClients = append(peerClients, peerClient)
-		}
-	}
-	return peerClients
-}
-
 // restClientFromHash will return a deterministic peerRESTClient based on s.
 // Will return nil if client is local.
 func (sys *NotificationSys) restClientFromHash(s string) (client *peerRESTClient) {

